docs/core/deploying/prepare-libraries-for-trimming.md

```diff
@@ -26,11 +26,13 @@ These instructions show how to enable and resolve static analysis warnings to pr
 
 Ensure you are using the .NET 6 SDK for these steps. They will not work correctly in previous versions.
 
-## Enable Roslyn analyzer
+### Set IsTrimmable
 
-Set `<EnableTrimAnalyzer>true</EnableTrimAnalyzer>` (in .NET 6+) in your library project. This will not have any effect on the output, but it will enable trim analysis during build via a Roslyn analyzer.
+Set `<IsTrimmable>true</IsTrimmable>` (in .NET 6+) in your library project. This will mark your assembly as "trimmable". Being trimmable means when your library is used in a trimmed application the assembly can have its unused members trimmed in the final output.
```

Comment:
What do people use for .NET 5 or .NET Core 3.1 (if available)?

Reply:
This functionality isn't available in 5 or earlier. It is only available in 6.